knowledge-base/common-icon-names-list.md

```diff
@@ -0,0 +1,456 @@
+---
+title: Icon names list
+description: Valid list of Icon names 
+type: how-to
+page_title: Icon names list
+slug: common-kb-icon-names
+position: 
+tags: 
+ticketid: 
+res_type: kb
+---
+
+## Description
+
+Since the `IconName` class is marked as obsolete and will soon be removed, below you will find a valid list of icon names to be hardcoded.
```

Comment:
Reword this to be future proof, something like "if you have used the obsolete `iconname` class, here is a list of icon names in its stead"

Also, tweak the titles and file names to reflect that. We don't want this to combat the main article for popularity, it is an upgrading resource that shold be obsolete in less than a year